Sage/carbon
# fix(multi-action-button, split-button): ensure screen reader commands can navigate menu popup

fix #7054

### Proposed behaviour


 
- First option in menu popup gains focus when menu is open. 
- Existing keyboard navigation Space, Enter and Arrow Up/Down should still work as expected. 
- Menu should close when no option has focus. 
- Menu popup is no longer in a portal.

### Current behaviour



- Screen reader users using VoiceOver commands cannot navigate into the menu popup. 
- Menu popup in in a portal. 

### Checklist




#### QA


### Additional context



### Testing instructions



Best tested in an example containing several focusable elements e.g.: "Sizes" story. 
- Navigate to toggle button and use VO commands Control+Option+Space to open menu. 
- Use Control+Option+Right to navigate menu options - see VO move onto next button on the page instead of the menu.
